src/api/python/setup.py

```diff
@@ -0,0 +1,88 @@
+import os
+import sys
+import platform
+import subprocess
+import multiprocessing
+from setuptools import setup
+from distutils.errors import LibError
+from distutils.command.build import build as _build
+from setuptools.command.develop import develop as _develop
+
+
+build_env = dict(os.environ)
+build_env['PYTHON'] = sys.executable
+build_env['CXXFLAGS'] = "-std=c++11"
+
+build_dir = os.path.abspath(os.path.dirname(__file__))
+
+if sys.platform == 'darwin':
+    library_file = "libz3.dylib"
+elif sys.platform == 'win32':
+    library_file = "libz3.dll"
+else:
+    library_file = "libz3.so"
+
+def _configure_z3():
```

Comment:
If you're going to build Z3 I'd highly recommend using the CMake build system instead of the old build system because IMHO it is the future of Z3 development and is what packagers should be using. Of course I would say this though because I wrote the CMake build system :)